lib/jekyll-import/importers/rss.rb

```diff
@@ -89,6 +93,26 @@ def self.write_rss_item(item, options)
           end
 
           f.puts output
+          if canonical_link
+            f.puts <<~HTML
+              <p>Originally posted on <a href="#{item.link}">#{URI.parse(item.link).host}</a>.</p>
+            HTML
+          end
+        end
+      end
+
+      private_class_method def self.get_tags(item, options)
+        explicit_tag = options["tag"]
+        return explicit_tag unless explicit_tag.nil? || explicit_tag.empty?
+
+        tags_reference = options["extract_tags"]
+        if tags_reference
+          tags_from_feed = item.instance_variable_get("@#{tags_reference}")
+          if tags_from_feed.is_a?(Array)
+            tags = tags_from_feed.map { |feed_tag| feed_tag.content.downcase }
+            tags.uniq!
+            return tags unless tags.empty?
+          end
         end
       end
```

Comment:
Here's how I would implement it doing away with nested conditionals.

```suggestion
      def self.get_tags(item, options)
        explicit_tag = options["tag"]
        return explicit_tag unless explicit_tag.nil? || explicit_tag.empty?

        tags_reference = options["extract_tags"]
        return unless tags_reference

        tags_from_feed = item.instance_variable_get("@#{tags_reference}")
        return unless tags_from_feed.is_a?(Array)

        tags = tags_from_feed.map { |feed_tag| feed_tag.content.downcase }
        tags.empty? ? nil : tags.tap(&:uniq!)
      end
      private_class_method :get_tags
```
We don't have test coverage on this. So, we'll have to take the code at face-value.